From b672a33d0c408bd6119cda48ef5fd2ed33d26d7f Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:14 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Remap ImsReasonInfo code 332 (403 sip forbidden) to 510
  Update CarrierConfig about USCC ERI file

https://github.com/haggertk/android_packages_apps_CarrierConfig/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: Ide41061b3f9bc0a8d52fc149cb632263418cd085
---
 assets/carrier_config_311220.xml | 1 +
 assets/carrier_config_311580.xml | 1 +
 assets/carrier_config_44010.xml  | 3 +++
 3 files changed, 5 insertions(+)

diff --git a/assets/carrier_config_311220.xml b/assets/carrier_config_311220.xml
index 77a7762..88ba99e 100644
--- a/assets/carrier_config_311220.xml
+++ b/assets/carrier_config_311220.xml
@@ -19,5 +19,6 @@
         <boolean name="enabledNotifyWapMMSC" value="true" />
         <boolean name="enableMultipartSMS" value="false" />
         <boolean name="show_apn_setting_cdma_bool" value="true" />
+        <string name="carrier_eri_file_name_string">eri_311220.xml</string>
     </carrier_config>
 </carrier_config_list>
diff --git a/assets/carrier_config_311580.xml b/assets/carrier_config_311580.xml
index 7a972ce..272eda1 100644
--- a/assets/carrier_config_311580.xml
+++ b/assets/carrier_config_311580.xml
@@ -19,6 +19,7 @@
         <boolean name="enabledNotifyWapMMSC" value="true" />
         <boolean name="enableMultipartSMS" value="false" />
         <boolean name="show_apn_setting_cdma_bool" value="true" />
+        <string name="carrier_eri_file_name_string">eri_311220.xml</string>
         <boolean name="support_swap_after_merge_bool" value="false" />
         <string name="carrier_eri_file_name_string">eri_311580.xml</string>
     </carrier_config>
diff --git a/assets/carrier_config_44010.xml b/assets/carrier_config_44010.xml
index 73b212e..8df2892 100644
--- a/assets/carrier_config_44010.xml
+++ b/assets/carrier_config_44010.xml
@@ -10,5 +10,8 @@
 <long name="message_expiration_time_long" value="3600000" />
 <boolean name="ignore_data_enabled_changed_for_video_calls" value="true" />
 <boolean name="allow_emergency_numbers_in_call_log_bool" value="true" />
+<string-array name="ims_reasoninfo_mapping_string_array" num="1">
+  <item value="332||510" />
+</string-array>
 </carrier_config>
 </carrier_config_list>
-- 
2.17.1

